### ISSUE TITLE
Missing Configuration Parameters for Zooz Zen20

### ISSUE THREAD

**Issue by daphatty** [AlCalzone/node-zwave-js#4806]
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within ZWaveJS2MQTT?

NO, my problem is NOT within ZWaveJS2MQTT

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

I received the below configuration parameters from Zooz tech support after inquiring about what I initially believed to be missing functionality. As it turns out, the ability to track Amps and Voltage via the Zen20 is turned off by default. In order to enable this functionality, the parameters listed below need to be configured.

In my case, I leveraged the Custom Configuration feature of ZwaveJS2Mqtt to test and enact these configuration changes. Figured I'd share the information Zooz provided me so that it can be included in the main config for this device.

> Electrical Current (A) Report Frequency
> Parameter 35: The number entered as value corresponds to the number of seconds. So if 30 is entered, the Strip will report electrical current values (Amperes) every 30 seconds.
> 
> Values: 0 – electrical current reports disabled (default); 30 – 2678400.
> 
> Size: 4 byte dec
> 
> Voltage Report (V) Frequency
> Parameter 36: The number entered as value corresponds to the number of seconds. So if 30 is entered, the Strip will report Voltage values (Volts) every 30 seconds.
> 
> Values: 0 – voltage reports disabled (default); 30 – 2678400.
> 
> Size: 4 byte dec

### Device information

Manufacturer: Zooz
Model name: Zen20

Github Page: https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x015d/zen20.json

### How are you using `node-zwave-js`?

- [ ] `zwavejs2mqtt` Docker image (latest)
- [ ] `zwavejs2mqtt` Docker image (dev)
- [ ] `zwavejs2mqtt` Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [X] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

version:
`node-zwave-js` branch:
`zwavejs2mqtt` branch:

### Did you change anything?

no

### Did this work before?

Don't know, this is a new device

### Attach Driver Logfile

2022-07-14 14:39:57.076 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 32.6 => 34.9
2022-07-14 14:39:57.173 INFO ZWAVE: Node 65: value updated: 50-0-value-66049 32.6 => 34.9
2022-07-14 14:39:57.765 INFO ZWAVE: Calling api sendCommand with args: [
{ nodeId: 65, commandClass: 112 },
'set',
[ 35, 30, 4, [length]: 3 ],
[length]: 3
]
2022-07-14 14:40:03.077 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 34.9 => 27.3
2022-07-14 14:40:03.106 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.170 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.272 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.338 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.385 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:03.479 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:04.672 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:04.787 INFO ZWAVE: Node 36: value updated: 50-0-value-66049 0 => 0
2022-07-14 14:40:04.901 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:04.961 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3
2022-07-14 14:40:05.030 INFO ZWAVE: Node 65: value updated: 50-2-value-66049 27.3 => 27.3

**Comment by daphatty** [comment]
To add to this, Zooz support has provided me with a full list of configuration parameters for the Zen20. It appears that ZwaveJS is missing parameters 28-36.

https://www.support.getzooz.com/kb/article/1024-zen20-power-strip-advanced-settings/

**Comment by alvinchen1** [comment]
Not quite comfortable doing a PR to this yet, but if anyone wants it, here's the updated JSON file (as a uploadable TXT file)
[zen20.json.txt](https://github.com/zwave-js/node-zwave-js/files/9203209/zen20.json.txt)
.

I also noticed both of my Zen20s are randomly not reporting information.  I am using Z-waveJStoMQTT so was able to manually set 35 and 36 and it started reporting (A) and (V).  But on some of my plugs, I am not getting any valid readings of W or kWh.  I tried to manually unset and set 28 - 33, but that didn't seem to resolve it.  Was wondering if anyone else with the Zen20s is experiencing no information on W and kWh before I also try to either pair it to another hub or contact Zooz Support.  (Home Assistant 7.8, HASSOS 8.4, Supervised install 2022.07.0 with Z-WaveJStoMQTT btw).
